src/uct/ib/mlx5/dv/ib_mlx5dv_md.c

```diff
@@ -592,6 +599,18 @@ uct_ib_mlx5_devx_mem_reg(uct_md_h uct_md, void *address, size_t length,
         goto err_memh_free;
     }
 
+    if ((md->flags & UCT_IB_MLX5_MD_FLAG_MKEY_BY_NAME_RESERVE) &&
+        (flags & UCT_MD_MEM_SYMMETRIC_RKEY)) {
```

Comment:
minor, I'd swap the conditions, as we expect that the first one will mostly be true

Reply:
fixed